jupyter-extension/jupyterlab_pachyderm/handlers.py

```diff
@@ -240,15 +267,47 @@ async def get(self, path):
         self._finish_model(model, location=False)
 
 
+# TODO: see about writing to/from config file
 class ConfigHandler(BaseHandler):
+    def config_response(self) -> bytes:
+        if not self.client:
+            return json.dumps({"cluster_status": "INVALID"})
+
+        try:
+            self.client.auth.who_am_i()
+            cluster_status = "AUTH_ENABLED"
```

Comment:
An enum may be a bit overkill here, but do we want to make these constants to avoid semantic/value errors?

Reply:
good catch, i've removed those now.